src/main/kotlin/com/hedvig/accountservice/accounts/domain/MemberAccount.kt

```diff
@@ -268,7 +269,9 @@ class MemberAccount() {
         return entries
             .filter { entry -> entry.type == EntryType.LOSS }
             .filter { entry -> entry.fromDate.isAfter(period.atEndOfMonth()) }
-            .fold(Money.of(BigDecimal.ZERO, "SEK"), { sum, entry -> sum.add(entry.amount)})
+            .map { it.amount }
+            .ifEmpty { listOf(Money.of(BigDecimal.ZERO, "SEK")) }
```

Comment:
```suggestion
            .ifEmpty { listOf(Money.of(BigDecimal.ZERO, getCurrency())) }
```

And then do something like this
```
fun getCurrency() = entries.map { entry -> entry.amount.currency }.distinct().first()
``
`